08_analysis_tools/CalibrateVaneOffsets.m

```diff
@@ -0,0 +1,119 @@
+% / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / /
+% Calibrate Airflow Vane Mounting Offsets
+% / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / /
+
+% !!-- This script must be run *after running the logconv.m script. --!!
+
+% NOTE: airspeed scale, tube pressure drop, and vane position mapping
+%       calibrations are assumed completed before running this script.
+
+% Nonlinear least squares solution:
+% min_x ||f(x,u)||
+% subject to
+% w_d = 0, mean slip = 0
+% where 
+% y = f(x,u)
+% y_ref = [vI_n, vI_e, vI_d, slip]
+% x = [b_aoa, b_slip, w_n, w_e]
+% u = [u, aoa, slip, phi, theta, psi]
+
+%% / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / / /
+% Calibration data should satisfy the following criteria:
+% - in air at nominal airspeed
+% - constant altitude
+% - executing "figure-8" with minimal bank for at least 2 cycles (hint: use
+%   stabilized modes for collecting this data) -- note both left and right
+%   turns are needed to make the assumption that average slip in nominal
+%   conditions is = zero.
+% - wind speeds are assumed to be constant within the seleted data
+% - no vertical wind is present (assumed zero)
+% NOTE: slip will be assumed = zero in nominal stabilized conditions, this
+% may not be true in reality, but with wind, the mounting offset is
+% otherwise unobservable.
+
+% start and end times (modify these)
+t_st_cal = 2115;
```

Comment:
Can you set those times to the standard values?

Reply:
oops - yes, meant to do that.